utils/build/docker/python/django/django.app.urls.py

```diff
@@ -434,6 +434,7 @@ def view_iast_source_parameter(request):
     return HttpResponse("OK")
 
 
+@csrf_exempt
```

Comment:
If the default logic of Django for CSRF triggers header injection, you should fix ddtrace to not trigger it.

Reply:
No, it's not related to header injection, we should skip csrf_exempt in POST request due to extra validations and tokens that Django needs